src/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useRef } from "react";
+
+const InputWithLabel = ({ todoTitle, handleTitleChange, children }) => {
```

Comment:
Glad to see that you're still using destructuring here! Techniques like this will make your app higher in quality and easier to maintain in the future.